world_coord_controller:
  ros__parameters:
    velocity_ratio:
      x: 0.3
      y: 0.3
      z: 0.2
      psi: 0.3

joint_state_publisher:
  ros__parameters:
    rate: 50
    source_list: ['manual_joint']

joy_node:
  ros__parameters:
    autorepeat_rate: 50.0  # [Hz]
    # coalesce_interval_ms: 20  # [ms]

hand_tool_manager:
  ros__parameters:
    max_speed: 5.0
    pid:
      Kp: 5.0
      Ki: 0.0
      Kd: 0.0